website/content/en/docs/getting-started/getting-started-with-karpenter/karpenter-capacity-dashboard.json

```diff
@@ -1,1440 +1,1743 @@
+				"sort": 1,
+				"type": "query"
+			},
+			{
+				"current": {
+					"selected": true,
+					"text": [
+						"All"
+					],
+					"value": [
+						"$__all"
+					]
+				},
+				"datasource": {
+					"type": "prometheus",
+					"uid": "prometheus"
+				},
+				"definition": "label_values(karpenter_disruption_actions_performed_total,method)",
+				"hide": 0,
+				"includeAll": true,
+				"multi": true,
+				"name": "method",
+				"options": [],
+				"query": {
+					"qryType": 1,
+					"query": "label_values(karpenter_disruption_actions_performed_total,method)",
+					"refId": "PrometheusVariableQueryEditor-VariableQuery"
+				},
+				"refresh": 2,
+				"regex": "",
+				"skipUrlSync": false,
+				"sort": 1,
+				"type": "query"
+			},
+			{
+				"current": {
+					"selected": false,
+					"text": "nodepool",
+					"value": "nodepool"
+				},
+				"hide": 0,
+				"includeAll": false,
+				"multi": false,
+				"name": "distribution_filter",
+				"options": [
+					{
+						"selected": false,
+						"text": "arch",
+						"value": "arch"
+					},
+					{
+						"selected": false,
+						"text": "capacity_type",
+						"value": "capacity_type"
+					},
+					{
+						"selected": false,
+						"text": "instance_type",
+						"value": "instance_type"
+					},
+					{
+						"selected": false,
+						"text": "method",
+						"value": "method"
+					},
+					{
+						"selected": false,
+						"text": "namespace",
+						"value": "namespace"
+					},
+					{
+						"selected": false,
+						"text": "node",
+						"value": "node"
+					},
+					{
+						"selected": true,
+						"text": "nodepool",
+						"value": "nodepool"
+					},
+					{
+						"selected": false,
+						"text": "zone",
+						"value": "zone"
+					}
+				],
+				"query": "arch,capacity_type,instance_type,method,namespace,node,nodepool,zone",
+				"queryValue": "",
+				"skipUrlSync": false,
+				"type": "custom"
+			}
+		]
+	},
+	"time": {
+		"from": "now-6h",
+		"to": "now"
+	},
+	"timepicker": {},
+	"timezone": "",
+	"title": "Karpenter Capacity",
+	"uid": "ta8I9Q67z",
+	"version": 5,
+	"weekStart": "",
+	"Transform": "AWS::Serverless-2016-10-31",
+	"Resources": {
+		"Api": {
+			"Type": "AWS::Serverless::Api",
+			"Properties": {
+				"Name": {
+					"Fn::Sub": [
+						"${ResourceName} From Stack ${AWS::StackName}",
+						{
+							"ResourceName": "Api"
+						}
+					]
+				},
+				"StageName": "Prod",
+				"DefinitionBody": {
+					"openapi": "3.0",
+					"info": {},
+					"paths": {
+						"/": {
+							"get": {
+								"responses": {}
+							}
+						}
+					}
+				},
+				"EndpointConfiguration": "REGIONAL",
+				"TracingEnabled": true
+			}
+		},
+		"UserPool": {
+			"Type": "AWS::Cognito::UserPool",
+			"Properties": {
+				"AdminCreateUserConfig": {
+					"AllowAdminCreateUserOnly": false
+				},
+				"AliasAttributes": [
+					"email",
+					"preferred_username"
+				],
+				"UserPoolName": {
+					"Fn::Sub": "${AWS::StackName}-UserPool"
+				}
+			}
+		}
+	}
```

Comment:
I'm curious where this came from, seems like it was added in this latest commit. I can't figure out what, if anything, this affects in the dashboard but it seems out of place and looks more like cloudformation json than anything.

Reply:
That's really kind of strange. I am not aware how this happened, sorry for that. Removed it , should work now.